lumicks/pylake/kymo.py

```diff
@@ -133,6 +133,18 @@ def _timestamps(self, sample_timestamps):
         return reconstruct_image(sample_timestamps, self.infowave.data,
                                  self.pixels_per_line, reduce=np.mean).T
 
+    @property
+    def pixelsize_um(self):
+        return [axes["pixel size (nm)"] / 1000 for axes in self._ordered_axes()]
+
+    @property
+    def delta_t(self):
+        """Line time in seconds"""
```

Comment:
Why not call it `line_seconds` or `line_time_seconds`? `delta_t` could easily also refer to pixel time.

Reply:
`line_time_seconds` it is! 👍